apis/betterself/v1/tests/test_adapters.py

```diff
@@ -419,3 +418,25 @@ def setUpTestData(cls):
         default, _ = User.objects.get_or_create(username='default')
         SupplementEventFactory(user=default)
         ProductivityLogFixturesGenerator.create_fixtures(default)
+
+
+class UserActivityLogAdaptersTests(AdapterTests, TestResourceMixin):
+    model = UserActivity
+
+    @classmethod
+    def setUpTestData(cls):
+        super().setUpTestData()
+
+        default, _ = User.objects.get_or_create(username='default')
+        UserActivityFactory(user=default)
+
+
+class UserActivityEventLogAdaptersTests(AdapterTests, TestResourceMixin):
```

Comment:
This was too easy to pass ... 

You should probably have more integration tests (even if they run slow) that do POST/DELETE, etc